modules/340-extended-monitoring/openapi/config-values.yaml

```diff
@@ -46,6 +46,28 @@ properties:
                 type: string
                 description: |
                   Root CA certificate to validate the container registry's HTTPS certificate (if self-signed certificates are used).
+      forceCheckDisabledControllers:
+        type: array
+        x-examples:
+          - ["Deployment", "StatefulSet"]
+        description: |
+          A list of controller kinds for which image is forcibly checked, even when workloads are disabled or suspended.
+          
+          The option is case-insensitive.
+          
+          Specify `*` to check all controller kinds.
+        items:
+          type: string
+          enum:
+            - Deployment
+            - StatefulSet
+            - DaemonSet
+            - CronJob
+            - deployment
+            - statefulset
+            - daemonset
+            - cronjob
```

Comment:
```suggestion
```
These enum values are not allowed by the validation. I assume this is fine because there are other options available.

Reply:
We have a small issue with Enum validation. According to [this](https://github.com/deckhouse/deckhouse/blob/90136124ad01d017a302752146660a72a822fa82/testing/openapi_validation/validators/enum.go#L267-L296) code in validator, the values should start with an uppercase letter. That means we cannot specify in the documentation that case-insensitive values can be used, nor can we use the `*` symbol.

Using `string` as the array element type is also not an option because, in that case, we cannot strictly limit the values that can be passed into the options array.

The only option I see is to change the Enum values, removing the values that start with a lowercase letter and replacing `*` with `All`.

Also, in the Helm template, it will be necessary to replace `All` with `*` and to modify the tests.

If this option works for you, I will make changes to the Pull Request.